doc/magic_cards_notes.md

```diff
@@ -1084,6 +1144,7 @@ hf 14a raw -s -c -t 1000 CF<passwd>34<1b length><0-16b ATS>
 Example: ATS to 0606757781028002F0, default pwd
 ```
 hf 14a raw -s -c -t 1000 CF000000003406067577810280
+script run hf_mf_ultimatecard -z 06067577810280
```

Comment:
I would definite clarify that the two commands isn't necessary to do it,  but it is alternatives.

Reply:
Done